recipes/Orange3-Bioinformatics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Orange3-Bioinformatics" %}
+{% set version = "3.0.3" %}
+{% set sha256 = "5dd3f52cb8ad4f4edddabc25e754e65d6a1616958e440a87c647e02f3098d98c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
We now [prefer `pip`]( https://conda-forge.org/docs/meta.html#use-pip ) for installs. Please update accordingly.